initial_pose:
  position: 
    x: 0.5
    y: 0.5
    z: 0.5
  orientation: 
    x: 0
    y: 0
    z: 0
    w: 1
